common/src/main/java/org/apache/rocketmq/common/UtilAll.java

```diff
@@ -199,18 +199,20 @@ public static double getDiskPartitionSpaceUsedPercent(final String path) {
         if (null == path || path.isEmpty())
             return -1;
 
+        String[] pathArr = path.trim().split(";");
         try {
```

Comment:
In the function,you should add some comments,such as if you path parameter is a mutiple-disk paths variable.